src/awkward/_v2/highlevel.py

```diff
@@ -2312,7 +2315,7 @@ def snapshot(self):
         """
         formstr, length, container = self._layout.to_buffers()
         form = ak._v2.forms.from_json(formstr)
-        return ak._v2.operations.from_buffers(form, length, container)
+        return ak._v2.operations.from_buffers(form, length, container, highlevel=True)
```

Comment:
`highlevel=True` by default (or at least it should be, for all functions that define a `highlevel` parameter).